CONTRIBUTING.rdoc

```diff
@@ -193,23 +193,23 @@ length for a comment, for example.
 
 ==== Directives
 
-For comments in markup you can add new directives (:nodoc: is a directive).
+For comments in markup you can add new directives (+:nodoc:+ is a directive).
 Directives may replace text or store it off for later use.
 
 See RDoc::Markup::PreProcess::register for details.
 
-=== JSONIndex
+=== JSON Index
 
-RDoc contains a special generator, RDoc::Generator::JSONIndex, which creates a
+RDoc contains a special generator, RDoc::Generator::JsonIndex, which creates a
 JSON-based search index and includes a search engine for use with HTML output.
 This generator can be used to add searching to any HTML output and is designed
 to be called from inside an HTML generator.
 
 == Markup
 
 Additional documentation markup formats can be added to RDoc.  A markup
-parsing class must respond to \::parse and accept a String argument containing
-the markup format.  An RDoc::Document containing documentation items
+parsing class must respond to +::parse+ and accept a String argument containing
+the markup format.  An RDoc::Markup::Document containing documentation items
 (RDoc::Markup::Heading, RDoc::Markup::Paragraph, RDoc::Markup::Verbatim, etc.)
```

Comment:
[note] `RDoc::Markup::Heading` exists but isn't rendered as a link in HTML. This seems like a bug, but I'm not sure about the details. 🤔 

https://github.com/ruby/rdoc/blob/fdaa5a605a93cb274556587b9152f94dff2b2fb1/lib/rdoc/markup/heading.rb#L5

<img width="294" alt="image" src="https://github.com/user-attachments/assets/efa6a123-996b-4839-a5e3-83ac60780583" />

Reply:
Similarly, `RDoc::Text::MARKUP_FORMAT` is also not rendered as a link, regardless of its existence.

https://github.com/ruby/rdoc/blob/19f05127bdacd87d5044fd261509c8a698c2b5a0/CONTRIBUTING.rdoc?plain=1#L217

https://github.com/ruby/rdoc/blob/fdaa5a605a93cb274556587b9152f94dff2b2fb1/lib/rdoc/text.rb#L23

<img width="300" alt="image" src="https://github.com/user-attachments/assets/95b882ef-4dd9-48ec-941c-05d1f2ae6dd7" />